File: examples/python/io/save_map_to_file.py
import os
import pywavemap as wave

# Create an empty map for illustration purposes
your_map = wave.Map.create({
    "type": "hashed_chunked_wavelet_octree",
    "min_cell_width": {
        "meters": 0.1
    }
})

# Save the map
user_home = os.path.expanduser('~')
map_path = os.path.join(user_home, "your_map.wvmp")
your_map.store(map_path)

File: examples/cpp/io/save_map_to_file.cc
#include <wavemap/io/file_conversions.h>

using namespace wavemap;  // NOLINT
int main(int, char**) {
  // Create an empty map for illustration purposes
  HashedWaveletOctreeConfig config;
  HashedWaveletOctree map(config);

  // Save the map
  const bool success = io::mapToFile(map, "/path/to/your/map.wvmp");
  CHECK(success);
}
